pymc3/diagnostics.py

```diff
@@ -141,6 +142,22 @@ def gelman_rubin(mtrace, varnames=None, include_transformed=False):
     Brooks and Gelman (1998)
     Gelman and Rubin (1992)"""
 
+    def rscore(x, num_samples):
+        # Calculate between-chain variance
+        B = num_samples * np.var(np.mean(x, axis=1), axis=0, ddof=1)
+
+        # Calculate within-chain variance
+        W = np.mean(np.var(x, axis=1, ddof=1), axis=0)
+
+        # Estimate of marginal posterior variance
+        Vhat = W * (num_samples - 1) / num_samples + B / num_samples
+
+        return np.sqrt(Vhat / W)
+
+    if not isinstance(mtrace,MultiTrace):
```

Comment:
Need space here `(mtrace, MultiTrace)` (PEP8)

Reply:
Thank you.